website/source/layouts/guides.erb

```diff
@@ -1,195 +1,88 @@
-          </li>
-          <li<%= sidebar_current("guides-upgrading-to-0.10.2") %>>
-            <a href="/guides/upgrading/upgrade-to-0.10.2.html">Upgrade to 0.10.2</a>
-          </li>
-          <li<%= sidebar_current("guides-upgrading-to-0.10.4") %>>
-            <a href="/guides/upgrading/upgrade-to-0.10.4.html">Upgrade to 0.10.4</a>
-          </li>
-          <li<%= sidebar_current("guides-upgrading-to-0.11.0") %>>
-            <a href="/guides/upgrading/upgrade-to-0.11.0.html">Upgrade to 0.11.0</a>
-          </li>
-        </ul>
-      </li>
+    <ul class="nav docs-nav">
+      <hashi-docs-sidebar
+        category='guides'
+        current_page='<%= current_page.path %>'
+        data="<%= encode(sidebar_data('guides')) %>"
+        order="<%= encode([
+          {
+            category: 'getting-started',
+          }, {
+            category: 'operations',
+            content: [
+              'reference-architecture',
+              'vault-ha-consul',
+              'production',
+              'generate-root',
+              'rekeying-and-rotating',
+              'plugin-backends',
+              '--------------',
+              'replication',
+              'disaster-recovery',
+              'mount-filter',
+              'multi-tenant',
+              'autounseal-aws-kms',
+              'seal-wrap',
+              'monitoring'
+            ]
+          }, {
+            category: 'identity',
+            content: [
+              'secure-intro',
+              'policies',
+              'authentication',
+              'approle-trusted-entities',
+              'lease',
+              'identity',
+              '--------------',
+              'sentinel',
+              'control-groups'
+            ]
+          }, {
+            category: 'secret-mgmt',
+            content: [
+              'static-secrets',
+              'versioned-kv',
+              'dynamic-secrets',
+              'db-root-rotation',
+              'cubbyhole',
+              'ssh-otp',
+              'pki-engine',
+              'app-integration'
+            ]
+          }, {
+            category: 'encryption',
+            content: [
+              'transit',
+              'spring-demo',
+              'transit-rewrap'
+            ]
+          }, {
+            category: 'upgrading',
+            content: [
+            'upgrade-to-0.5.0',
+            'upgrade-to-0.5.1',
+            'upgrade-to-0.6.0',
+            'upgrade-to-0.6.1',
+            'upgrade-to-0.6.2',
+            'upgrade-to-0.6.3',
+            'upgrade-to-0.6.4',
+            'upgrade-to-0.7.0',
+            'upgrade-to-0.8.0',
+            'upgrade-to-0.9.0',
+            'upgrade-to-0.9.1',
+            'upgrade-to-0.9.2',
+            'upgrade-to-0.9.3',
+            'upgrade-to-0.9.6',
+            'upgrade-to-0.10.0',
+            'upgrade-to-0.10.2',
+            'upgrade-to-0.10.4',
+            'upgrade-to-0.11.0'
```

Comment:
FWIW, IMHO the order of this list should be reversed. Newer upgrade guides should be at the top. (Normally I'd just submit a PR to fix, but I don't want to worsen the merge conflicts you're likely going to have to deal with.) My view is that the most relevant information should always be at the top. I'f I'm on a 0.10.0 install, I want to see all the upgrade notes from 0.10.0 to current, rather than have to scroll past the 15 upgrade guides that aren't relevant to me to even see if there are any that are relevant.

Reply:
That sounds entirely reasonable to me, but in this case I think I'd rather open that up as an issue and tackle if after this gets merged hah. Does that seem reasonable?

Reply:
Sounds good -- I'll wait until this is merged and then open a PR.